TheRocket/TheRocket.ipynb

```diff
@@ -10,6 +10,209 @@
     "- Adhere to units seen in comments [conversion done in 'Solver Run']"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read in data from spreadsheet at ./data/variables.xlsx\n",
+    "import pandas as pd\n",
+    "def readVars(sheet, skiprows, nrows):\n",
+    "    data = pd.read_excel('./data/variables.xlsx', sheet_name=sheet, skiprows = skiprows, nrows = nrows)\n",
+    "    # Variable \tValue\tUnit\tDescription\tFinalized\n",
+    "    data.columns = ['Variable', 'Value', 'Unit', 'Description', 'Finalized']\n",
+    "    print(data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "                    Variable  Value Unit  Description  Finalized\n",
+      "0           vehicle_mass_dry     79   lb          NaN        NaN\n",
+      "1  vehicle_COM_dry_no_engine     68   in          NaN        NaN\n",
+      "2           vehicle_diameter      6   in          NaN        NaN\n",
+      "3             vehicle_height    151   in          NaN        NaN\n",
+      "4                rail_length     20   ft          NaN        NaN\n",
+      "5               launch_angle      5  deg          NaN        NaN\n",
+      "             Variable  Value Unit  Description  Finalized\n",
+      "0      tank_height_ox   34.5   in          NaN        NaN\n",
+      "1    tank_height_fuel   10.3   in          NaN        NaN\n",
+      "2    tank_position_ox   71.0   in          NaN        NaN\n",
+      "3  tank_position_fuel   35.0   in          NaN        NaN\n",
+      "          Variable    Value        Unit  Description  Finalized\n",
+      "0  nosecone_height       24          in          NaN        NaN\n",
+      "1    nosecone_kind  \"ogive\"  use quotes          NaN        NaN\n",
+      "         Variable  Value Unit  Description  Finalized\n",
+      "0      fin_number    4.0  NaN          NaN        NaN\n",
+      "1  fin_root_chord    9.0   in          NaN        NaN\n",
+      "2   fin_tip_chord    2.5   in          NaN        NaN\n",
+      "3        fin_span    5.5   in          NaN        NaN\n",
+      "4    fin_position   14.0   in          NaN        NaN\n",
+      "                 Variable  Value Unit  Description  Finalized\n",
+      "0  button_position_higher   75.5   in          NaN        NaN\n",
+      "1   button_position_lower   15.1   in          NaN        NaN\n",
+      "2            button_angle   45.0  deg          NaN        NaN\n"
+     ]
+    }
+   ],
+   "source": [
+    "# STRUCTURES\n",
+    "readVars(\"STRUCTURES2\", 1, 6) # General\n",
```

Comment:
It looks like "STRUCTURES2" isn't a sheet in the excel file you included in the PR. Should this just be "STRUCTURES"?

Reply:
No worries at all - thanks sm